server/src/main/java/org/jboss/as/server/operations/NativeRemotingManagementAddHandler.java

```diff
@@ -70,7 +78,16 @@ public void performRuntime(OperationContext context, ModelNode operation, ModelN
         ManagementChannelRegistryService.addService(serviceTarget, endpointName);
         ManagementRemotingServices.installManagementChannelServices(serviceTarget,
                 endpointName,
-                new ModelControllerClientOperationHandlerFactoryService(),
+                new ModelControllerOperationHandlerFactory() {
```

Comment:
That'd change the API but why do we need to instantiate the service when `ManagementRemotingServices.installManagementChannelServices` is called? 
Could we instead instantiate the service *inside* `ManagementRemotingServices.installManagementChannelServices`?

Reply:
See my comment above from 8-th of October @jmesnil .

Reply:
After reading more carefully the code, I see now why you did it that way. thanks